scripts/dev/devimint/aliases/fedimint-cli

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+$FM_ALIAS_ORIG_FEDIMINT_CLI --data-dir "$FM_CLIENT_DIR" "$@"
```

Comment:
i don't like having to manually do aliasing instead of using devimint environment variables :/

Reply:
It's not like env variable shows up in the PATH magically, so something still needs to be done to actually make it usable.

Reply:
@maan2003 https://github.com/fedimint/fedimint/pull/6661